client/my-sites/media-library/style.scss

```diff
@@ -215,6 +216,10 @@
 	margin-bottom: 0;
 }
 
+.media-library__content {
+	min-height: 600px;
```

Comment:
You mentioned this hack in chat, and indeed this will need a different approach. This creates an additional scrollbar, so for starters we need to take this out. 

However you also noted the reason you added this was a clipped dropdown when there's little content, and sure enough:

![Screenshot 2019-05-28 at 08 56 02](https://user-images.githubusercontent.com/1204802/58457271-8dd81600-8126-11e9-9012-a09011d8fcc0.png)

The easy fix is to move the entire dropdown into the `media-library__list` container. This means it won't stay fixed at the top, but it's also an okay fix. 

If we did not use a custom select box, this would be a non issue.

Reply:
![Screenshot 2019-05-28 at 09 11 02](https://user-images.githubusercontent.com/1204802/58458082-89146180-8128-11e9-9c73-fa9d1e4cbb2c.png)

Moving the category dropdown to that list would make it look like the above. Which is not as ideal, but without rather involved refactor of the media library, or reverting to using a normal select box, I don't think there's an easy fix.